pkg/ipmi/ocp/bootorder.go

```diff
@@ -93,39 +87,45 @@ func remapSortBootOrder(BootOrder *BootOrder) {
 func updateVPDBootOrder(i *ipmi.IPMI, BootOrder *BootOrder) error {
 	var err error
 	var key string
-	var bootType byte
 	var idx int
-	for _, v := range BootOrder.bootSeq {
+	for _, bootType := range BootOrder.bootSeq {
 		key = fmt.Sprintf("Boot%04d", idx)
-		bootType = v & 0x7
 		if bootType == NETWORK_BOOT {
 			log.Printf("VPD set %s to %s", key, NETBOOTER_CONFIG)
 			BootEntries = append(BootEntries, systembooter.BootEntry{Name: key, Config: []byte(NETBOOTER_CONFIG)})
 			if err = Set(key, []byte(NETBOOTER_CONFIG)); err != nil {
 				return err
 			}
 			idx++
+		} else if bootType == NETWORK_BOOT_IPV6 {
+			log.Printf("VPD set %s to %s", key, NETBOOTER_IPV6_CONFIG)
+			BootEntries = append(BootEntries, systembooter.BootEntry{Name: key, Config: []byte(NETBOOTER_IPV6_CONFIG)})
+			if err = Set(key, []byte(NETBOOTER_IPV6_CONFIG)); err != nil {
```

Comment:
Rebasing to the current upstream this needs to be changed to
vpd.FlashromRWVpdSet(key, []byte(NETBOOTER_IPV6_CONFIG), false);

Reply:
ack